app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyQuestionScreen.kt

```diff
@@ -41,10 +41,14 @@ fun SurveyQuestionScreen(
         )
         SurveyBoldText(text = surveyQuestion.title, fontSize = 34.sp)
         Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
-            if (surveyQuestion.questionDisplayType == QuestionDisplayType.DROPDOWN) {
-                SurveyDropDownQuestion(answers = surveyQuestion.answers) {
+            when (surveyQuestion.questionDisplayType) {
+                QuestionDisplayType.DROPDOWN -> SurveyDropDownQuestion(answers = surveyQuestion.answers) {
                     onChooseAnswer(surveyQuestion.id, it)
                 }
+                QuestionDisplayType.SMILEY -> SurveySmileyQuestion()
```

Comment:
How about adding an import for these `QuestionDisplayType`s? 😄 
```suggestion
                DROPDOWN -> SurveyDropDownQuestion(answers = surveyQuestion.answers) {
                    onChooseAnswer(surveyQuestion.id, it)
                }
                SMILEY -> SurveySmileyQuestion()
```